backlog.md

```diff
@@ -11,4 +11,10 @@ Suggestions of other unit tests:
 What if the allowed_choices is an empty list
 What if the allowed_choices is a list of something that's not strings (e.g. ints, floats, bools)
 What if the allowed_choices is not a list (e.g. it's None, or a single string, or a single number)
-What if the user types a number
\ No newline at end of file
+What if the user types a number
+
+Update readme.md
+  specift python version
+  give list of packages used
+
+Edgecase in define_allowed_direction where ant has food 'behind' (eg) it but those food have ants on them so they are not accessible.
```

Comment:
If this has already been done, remove from the backlog or move to a "completed" section